x-pack/plugins/security_solution/server/lib/telemetry/types.ts

```diff
@@ -138,3 +141,35 @@ interface EndpointMetricOS {
   platform: string;
   full: string;
 }
+
+export interface ESLicense {
+  status: string;
+  uid: string;
+  type: string;
+  issue_date?: string;
+  issue_date_in_millis?: number;
+  expiry_date?: string;
+  expirty_date_in_millis?: number;
+  max_nodes?: number;
+  issued_to?: string;
+  issuer?: string;
+  start_date_in_millis?: number;
+}
+
+export interface TelemetryEvent {
```

Comment:
Good idea moving to migrate this to `types.ts`

What you have done though is import this into the sender, and not re-export (the right thing to do), but you need to update your references to this object. 

```
diagnostic_task.ts
~~~

import { TelemetryEventsSender, TelemetryEvent } from './sender';
=> 
import { TelemetryEvent } from './types
```

You are going to have to update the references in:

  - x-pack/plugins/security_solution/server/lib/detection_engine/signals/send_telemetry_events.ts:8:33 
 
Same with your `copyAllowlistedFields` reference in 

  - x-pack/plugins/security_solution/server/lib/telemetry/sender.test.ts:9:33